Makefile

```
@@ -40,9 +40,17 @@ install-markdownlint:
 markdownlint:
 	@for f in $(ALL_DOCS); do echo $$f; $(MARKDOWN_LINT) -c .markdownlint.yaml $$f || exit 1;	done
 
+# Generate markdown tables from YAML definitions
 .PHONY: table-generation
 table-generation:
-	docker run --rm -v $(PWD)/semantic_conventions:/source -v $(PWD)/specification:/spec otel/semconvgen -f /source markdown -md /spec
+	docker run --rm -v $(PWD)/semantic_conventions:/source -v $(PWD)/specification:/spec \
+		otel/semconvgen:$(SEMCONVGEN_VERSION) -f /source markdown -md /spec
+
+# Check if current markdown tables differ from the ones that would be generated from YAML definitions
+.PHONY: table-check
+table-check:
```

```suggestion
check-tables:
```

IMHO we should also change table-generation to generate-tables. We could keep the old name for compatibility.

---

I think `table-generation` and `table-check` is fine, it also works well with bash completion.
We also have `markdown-link-check` and _not_ `check-markdown-links`, for example, so we would have to change more targets to be consistent. Same for `misspell-correction`.